src/main/java/com/ericsson/eiffel/remrem/semantics/SemanticsService.java

```diff
@@ -92,4 +93,32 @@ public String getEventId(JsonObject json) {
         }
         return null;
     }
+
+    @Override
+    public String getFamily(JsonObject eiffelMessage) {
+        return null;
+    }
+    
+    @Override
+    public String getType(JsonObject eiffelMessage) {
+        return null;
+    }
+    @Override
+    public String getServiceName() {
+        return "eiffel-semantics";
```

Comment:
This is intended for the service name so is should be eiffelsemantics as in eiffel-remrem-generate today. Otherwise we will make an incompatible change.